tests/providers/amazon/aws/log/test_s3_task_handler.py

```diff
@@ -107,9 +106,8 @@ def test_hook_raises(self):
 
             mock_error.assert_called_once_with(
                 'Could not create an S3Hook with connection id "%s". Please make '
-                'sure that apache-airflow[aws] is installed and the S3 connection exists. Exception : "%s"',
```

Comment:
This an example of where I'm unsure if accounting for `G200` error (Logging statements should not include the exception in logged string (use exception or exc_info=True) will negatively affect what's logged and, I suppose more importantly, what the log handlers will ingest. 

Here the "Exception: <exception message>" is part of the logging line directly. But does this matter when `exc_info=True` or when there is a following `raise`? Is there really any lost information in the logs?

Reply:
An option could be ignoring the `G200` error as well as part of `flake8-logging-format` should we want to introduce it. @kaxil Do you have an opinion here since you'd taken a crack at using this flake8 plugin elsewhere?